docs/user-guide/services/index.md

```diff
@@ -147,11 +147,38 @@ Accessing a `Service` without a selector works the same as if it had selector.
 The traffic will be routed to endpoints defined by the user (`1.2.3.4:9376` in
 this example).
 
+A special case of service without selectors is an ExternalName service.
+It does not define any ports or endpoints. Rather, it serves as a way
+to return an alias to an external service residing outside the cluster.
+
+```json
+{
+    "kind": "Service",
+    "apiVersion": "v1",
+    "metadata": {
+        "name": "my-service",
+        "namespace": "prod"
+    },
+    "spec": {
+        "type": "ExternalName",
+        "externalName": "my.database.example.com"
+    }
+}
+```
+
+When looking up the host `my-service.prod.svc.CLUSTER`, the cluster DNS service
+will return a `CNAME` record with the value `my.database.example.com`. Accessing
+such a service works in the same way as others, with the only difference that
+the redirection happens at the DNS level and no proxying or forwarding occurs.
+Should you later decide to move your database into your cluster, you can start
+its pods, add appropriate selectors or endpoints and change the service `type`.
+
 ## Virtual IPs and service proxies
 
-Every node in a Kubernetes cluster runs a `kube-proxy`.  This application
-is responsible for implementing a form of virtual IP for `Service`s.  In
-Kubernetes v1.0 the proxy was purely in userspace.  In Kubernetes v1.1 an
+Every node in a Kubernetes cluster runs a `kube-proxy`.  This application is
```

Comment:
Pronoun avoidance.

"Every node in a Kubernetes cluster runs a `kube-proxy`. `kube-proxy` is responsible for..."